source/mesh_deformation/interface.cc

```diff
@@ -210,17 +368,31 @@ namespace aspect
                            "\n\n"
                            "The format is id1: object1 \\& object2, id2: object3 \\& object2, where "
                            "objects are one of " + std::get<dim>(registered_plugins).get_description_string());
+        prm.declare_entry("Surface stabilization theta", "0.5",
```

Comment:
You are moving this parameter into a different subsection, which is not backwards compatible. I think we need to deprecate the old value or keep it in the old section.

Reply:
Okay, so if I understand correctly one option would be to move this back to being declared in free_surface.cc and then have the interface parse into that to get the value. Should I change the variable name back into "Free surface stabilization theta" in that case?

What would be the preferred way, keeping it in the old section or deprecating it? I am not sure the process for deprecating a value.

Reply:
No, you can leave it in the "free surface" subsection by entering the right subsection right here. No need to move this to the other file.